string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,30 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  string = string.toLowerCase();
+  var newString = '';
+  var partString = '';
+  partString = partString + string.charAt(0).toUpperCase();
+  for (var i = 1; i < string.length; i++) {
+    if (string[i - 1] === ' ') {
+      partString = partString + string.charAt(i).toUpperCase();
+    } if (string[i] !== ' ' & string[i - 1] !== ' ') {
+      partString = partString + string[i];
+    } if (string[i] === ' ') {
+      partString = partString + ' ';
+      newString = newString + partString;
+      partString = '';
+    } if (i === (string.length - 1)) {
+      newString = newString + partString;
```

Comment:
Are these meant to be `else if` statements? They're formatted as if they are meant to be, but they are all just independent `if` statements currently.

Reply:
Hey Cody! I actually meant for them to be independent if statements.  Is that bad practice to do so?